app/src/main/res/layout/dialog_help.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+<LinearLayout 
     android:layout_width="match_parent"
     android:layout_height="match_parent"
```

Comment:
Replace the height 'match_parent' with 'wrap_content'

Reply:
OK, I will change that. :) Thank you so much!